build/cmake/functions.cmake

```diff
@@ -342,22 +351,23 @@ function(wx_set_target_properties target_name is_base)
     if(wxBUILD_SHARED)
         string(TOUPPER ${target_name_short} target_name_upper)
         if(target_name_short STREQUAL "mono")
-            target_compile_definitions(${target_name} PRIVATE DLL_EXPORTS WXMAKINGDLL)
-        else()
-            target_compile_definitions(${target_name} PRIVATE DLL_EXPORTS WXMAKINGDLL_${target_name_upper})
```

Comment:
Why we don't need `DLL_EXPORTS` anymore?

Reply:
wxWidgets does not use it. And things like `__declspec(dllexport)` are covered via `WXMAKINGDLL_*`, see `wx/dlimpexp.h`.